app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -82,7 +82,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var text = data.text || event.target.textContent
       data.text = window.GOVUK.analyticsGa4.core.trackFunctions.removeLinesAndExtraSpaces(text)
       data.text = this.PIIRemover.stripPIIWithOverride(data.text, true, true)
-      if (!data.text && (element.querySelector('img') || element.querySelector('svg'))) {
+      if (!data.text && (element.querySelector('img') || element.querySelector('svg') || element.closest('img') || element.closest('svg'))) {
```

Comment:
Correct, I guess the reason I used `closest` is because it also checks the current element itself as well as any children. I could make it `element.tagName === 'IMG'` instead?

Reply:
Ah! Things I learned today - `closest` includes the element itself. Don't think I knew that 👍 

Yes, I think changing it to check the tagName is probably clearer (and might be fractionally more efficient - if it can't find an `img` it might traverse all the way up to the top level before stopping), good idea 👍

Reply:
Thanks, done! :+1: